db/seeds.rb

```diff
@@ -1,30 +1,43 @@
-FactoryBot.create(:user, email: 'admin@admin.com', password: '1qaz2wsx', role: 'admin')
-FactoryBot.create(:user, email: 'super_admin@super_admin.com', password: '1qaz2wsx', role: 'super_admin')
-FactoryBot.create(:user, email: 'sale@sale.com', password: '1qaz2wsx', role: 'sale')
+if Rails.env.production?
+  Tenant.create(name: "super")
+  Tenant.create(name: "admin")
+  Tenant.create(name: "sale")
+  User.create(first_name: "super", last_name: "puper", email: "egor.odessa@gmail.com", role: "super_admin", password: "superteam", tenant: Tenant.find_by(name: "super"))
```

Comment:
instead of `Tenant.find_by(name: "super")` just make it work with local var for tenant
```
tenant = Tenant.create(name: 'AwesomeTenant')
User.create(tenant: tenant)
```